src/openrct2/interface/Window_internal.cpp

```diff
@@ -4,13 +4,13 @@
 
 void rct_window::SetLocation(int32_t newX, int32_t newY, int32_t newZ)
 {
-    window_scroll_to_location(this, newX, newY, newZ);
+    window_scroll_to_location(this, { newX, newY, newZ });
     flags &= ~WF_SCROLLING_TO_LOCATION;
 }
 
 void rct_window::ScrollToViewport()
 {
-    int32_t newX = LOCATION_NULL, newY = LOCATION_NULL, newZ = LOCATION_NULL;
+    auto newCoords = CoordsXYZ{ LOCATION_NULL, LOCATION_NULL, LOCATION_NULL };
```

Comment:
These values seem to get always overwritten. It's better to initialise it like so:
```suggestion
    CoordsXYZ newCoords = {};
```